### ISSUE TITLE
s/compote/methylCC/g;

### ISSUE THREAD

**Issue by ttriche** [stephaniehicks/methylCC#1]
Vignette was confusing for a moment ("where is this compote you speak of?") but then it clicked...

How come objects must be eSet-derived instead of SummarizedExperiment-derived?  Much larger datasets can be handled with the latter (e.g. BSseq objects subclass SE, if memory serves, and Peter Hickey is merrily bolting on out-of-core storage for them).

**Comment by stephaniehicks** [comment]
In full honestly, I changed the name last night so I did a quick overhaul. I plan to fully revamp things next week. Also, I completely agree about SE object vs eSet-derived. I originally wrote that code over a year ago and I have been meaning to change that. It's on my to-do list for next week. I'll leave the issue here until I fix both these things, but just wanted to give a quick explanation. Thanks for the feedback though!
